README.md

```diff
@@ -1 +1,127 @@
-# REACT_SOLO_CAPSTONE
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  
+  <br/>
+
+  <h3><b>Welcome...</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+## 📖 [PREMIER LEAGUE] <a name="about-project"></a>
+
+My React project in Microverse.
+
+**[REACT_SOLO_CAPSTONE]** is a project experience that Microverse offers to help nuture your skills in designing application based on API data.  
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- React
+- Webpack
+- Jest
+
+
+### Key Features <a name="key-features"></a>
+
+ 1. Setup with create React app
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+In order to run this project you need:
+
+An extension to your browser
+### Setup
+
+Clone this repository to your desired folder:
+
+git clone [PREMIER LEAGUE](https://github.com/2004-okari/REACT_SOLO_CAPSTONE)
+### Install
+
+Install this project with:
+
+- `cd REACT_SOLO_CAPSTONE`
+- `git clone https://github.com/2004-okari/REACT_SOLO_CAPSTONE.git`
+  
+### Usage
+
+To run the project, execute the following command:
+Open the index.html file on the browser or use a VS code extension server to start the project`
+
+- run `npm install` or `npm i` in your terminal to download all the packages needed to make the project run
+- run `npm run start` or `npm start` to open the project in your browser
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Deployment
+
+[Sites link](https://thriving-daifuku-067d8b.netlify.app/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### 👤 Authors <a name="authors"></a>
+
```

Comment:
- [ ] As stated in the project requirement, you need to make a video that follows the guidelines listed there and as seen in this screenshot below, and then add the link to your readme, currently, you do not have that. i'll suggest you also work on this. :+1: :smile: 

- - Screenshot:
![Screenshot from 2023-09-06 18-59-52](https://github.com/2004-okari/REACT_SOLO_CAPSTONE/assets/107632780/fab350a1-f903-4637-96fe-7ae9e61d5590)